src/context/DatastoreReducer.js

```diff
@@ -18,7 +18,7 @@ const updateTimestamp = () => {
 
 const createLabel = (measure, info) => {
   if (info[measure]) {
-    return info[measure].displayLabel;
+    return info[measure].title;
```

Comment:
I think you want to combine these two. `${info[measure].displayLabel} - ${info[measure].title}`.